pagopa/io-app
# feat(Bonus Pagamenti Digitali): [#176072451] Added "share you screenshot" checkbox to "Write to the IO team" screen

## Short description
A checkbox to express one's will to also attach a screenshot has been added to the "Write to the IO team" screen. 

## How to test
- Click the ? sign on almost any screen
- Scroll through the text and click "Write to the IO team"
- Scroll through the end and see the checkbox

<img width="427" alt="Schermata 2020-12-09 alle 19 51 02" src="https://user-images.githubusercontent.com/26572302/101673900-10e66600-3a58-11eb-9209-adfb9528524e.png">
